solution/ui/regulations/utilities/api.js

```diff
@@ -100,7 +105,11 @@ function fetchJson(url, options = {}, retryCount = 0) {
     }
 
     return Promise.resolve()
-        .then(() => localforage.getItem(url.replace(apiPath, merged.method)))
+        .then(
+            () =>
+                cacheResponse &&
+                localforage.getItem(url.replace(apiPath, merged.method))
+        )
         .then((value) => {
             if (value && Date.now() < value.expiration_date) {
```

Comment:
if `cacheResponse` is `falsy`, `value` will be `undefined` and it will short circuit this `if` condition immediately and drop us into the `else` block, which will fetch from the server.